### PR TITLE
Calculate bench client lamports based on signature fee

### LINKED ISSUE #4695: Running bench-tps on the testnet runs the accounts out of tokens
#### Problem

bench-tps doesn't run successfully on the testnet and gives lots of AccountNotFound errors.

#### Proposed Solution

Debug and fix it.

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
The bench client runs out of tokens due to signature fee.

#### Summary of Changes
Increased the client account balance.

Fixes #4695

### CODE REVIEW

File: bench-tps/src/bench.rs [hunk 1]
```diff
@@ -25,7 +25,8 @@ use std::time::Duration;
 use std::time::Instant;
 
 pub const MAX_SPENDS_PER_TX: u64 = 4;
-pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 128;
+/// Enough tokens to run bench client for ~1 week at ~100K TPS
+pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 65536;
```
Comment (sakridge) [inline bench-tps/src/bench.rs]: thats a lot!

Comment (pgarg66) [reply bench-tps/src/bench.rs]: My calculations:
For 42 lamports per signautre, 100k TPS, 1 million accounts, and 1 week of runtime,
Total lamports required: 42 * 100,000 * 60 * 24 * 7 = 42,336,000,000
Divided by 1million accounts = 42,336
Rounded to next ^2

Comment (pgarg66) [reply bench-tps/src/bench.rs]: Actually, we may need even more. Forgot to multiply by 60 for number of minutes in one hour..

Comment (mvines) [reply bench-tps/src/bench.rs]: Can we change this from a compile-time constant to a runtime calculation based on the worse case signature fee/?   The current value of `42` will certainly change to X in the future

Comment (pgarg66) [reply bench-tps/src/bench.rs]: Yeah, that's a good idea.

Comment (pgarg66) [reply bench-tps/src/bench.rs]: is `--target-lamports-per-signature` the worst case fee? When the accounts are created (in bench-tps) we currently don't have access to the fee. We may need to redesign it a bit.

Comment (mvines) [reply bench-tps/src/bench.rs]: Use https://github.com/solana-labs/solana/blob/44967abd1c129e57824ec7fa8e2529bfb141584f/sdk/src/fee_calculator.rs#L23, which can be fetched via https://github.com/solana-labs/solana/blob/44967abd1c129e57824ec7fa8e2529bfb141584f/client/src/rpc_client.rs#L298

Comment (pgarg66) [reply bench-tps/src/bench.rs]: yes, but the bench-tps is run (to create and assign lamports to accounts) even before the bootstrap leader is started.
https://github.com/solana-labs/solana/blob/44967abd1c129e57824ec7fa8e2529bfb141584f/net/remote/remote-node.sh#L122

Comment (mvines) [reply bench-tps/src/bench.rs]: doh!   I guess we need to pass in the fee as a command-line argument 🤕

Comment (pgarg66) [reply bench-tps/src/bench.rs]: yeah, we could pass the genesis options to the bench-tps. But the option may not always be present. So the default value needs to be moved out of setup.sh..

**Pull request merged**